Header header
uint32 image_width
uint32 image_height
float64 angle_of_view
uint32 tag_count
Tag[] tags
